components/ContainerFull.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View } from 'react-native';
+import styles from './styles';
+
+interface Props {
+  internalComponent: any;
+}
+
+export default function ContainerFull({ internalComponent }: Props) {
```

Comment:
there's no need to pass components like that through props, it's better to use `children`:
```ts
export default function ContainerFull() {
  return <View style={styles.container}>{this.props.children}</View>;
}

// usage
<ContainerFull>
  <PutYourComponentHere />
</ContainerFull>
```

There should be no need to add `children` to `Props` as well -> it's always available (but if you'd have problems with TS please let me know)